.claude/hooks/pre_git_quality_gates.py

```
@@ -0,0 +1,149 @@
+            print(f"  ✅ 成功", file=sys.stderr)
+
+    except subprocess.TimeoutExpired:
+        failed_checks.append({
+            "name": check["name"],
+            "error": "タイムアウト (5分)"
+        })
+        print(f"  ❌ タイムアウト", file=sys.stderr)
+
+    except FileNotFoundError:
+        # スクリプトが存在しない場合はスキップ
+        print(f"  ⚠️  スキップ (コマンドが見つかりません)", file=sys.stderr)
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)